api/views.py

```diff
@@ -438,16 +442,18 @@ def all_activity_over_time(self, request, format=None):
             )
         ).filter(instructors=0).values_list('slug', flat=True)
 
-        return Response({
+        return {
             'start': start,
             'end': end,
             'workshops': {
                 'SWC': swc_workshops.count(),
                 'DC': dc_workshops.count(),
                 'SWC,DC': swc_dc_workshops,
+                'SWC_or_DC': swc_dc_workshops,
                 'WiSE': wise_workshops,
                 'TTT': ttt_workshops,
                 'self-organized': self_organized_workshops,
+                'self_organized': self_organized_workshops,
```

Comment:
One small final bit: can you please add a comment here explaining why we need `SWC,DC` and `SWC_or_DC` (same for self organized)?